EvidenceApi.Tests/V1/UseCase/FindDocumentSubmissionsByResidentIdUseCaseTests.cs

```diff
@@ -0,0 +1,149 @@
+using System;
+using AutoFixture;
+using EvidenceApi.V1.Boundary.Response.Exceptions;
+using EvidenceApi.V1.Boundary.Request;
+using EvidenceApi.V1.Domain;
+using EvidenceApi.V1.Gateways.Interfaces;
+using EvidenceApi.V1.UseCase;
+using FluentAssertions;
+using Moq;
+using NUnit.Framework;
+using System.Threading.Tasks;
+using EvidenceApi.V1.Boundary.Response;
+using System.Collections.Generic;
+
+namespace EvidenceApi.Tests.V1.UseCase
+{
+    public class FindDocumentSubmissionsByResidentIdUseCaseTests
+    {
+        private FindDocumentSubmissionsByResidentIdUseCase _classUnderTest;
+        private Mock<IEvidenceGateway> _evidenceGateway;
+        private Mock<IDocumentTypeGateway> _documentTypesGateway;
+        private Mock<IDocumentsApiGateway> _documentsApiGateway;
+        private readonly IFixture _fixture = new Fixture();
+        private DocumentType _documentType;
+        private EvidenceRequest _evidenceRequest1;
+        private EvidenceRequest _evidenceRequest2;
+        private DocumentSubmissionSearchQuery _useCaseRequest;
+        private DocumentSubmission _documentSubmission1;
+        private DocumentSubmission _documentSubmission2;
+        private List<DocumentSubmission> _found;
+        private Task<Claim> _claim1;
+        private Task<Claim> _claim2;
+        private string _claimId1 = "70cdff29-84d3-461e-bd16-2032c07c28bd";
+        private string _claimId2 = "010f4156-92aa-4082-891b-3b238e46940a";
+
+
+        [SetUp]
+        public void SetUp()
+        {
+            _evidenceGateway = new Mock<IEvidenceGateway>();
+            _documentTypesGateway = new Mock<IDocumentTypeGateway>();
+            _documentsApiGateway = new Mock<IDocumentsApiGateway>();
+            _classUnderTest = new FindDocumentSubmissionsByResidentIdUseCase(
+                _evidenceGateway.Object,
+                _documentTypesGateway.Object,
+                _documentsApiGateway.Object
+            );
+        }
+
+        [Test]
+        public async Task ReturnsTheFoundDocumentSubmissions()
+        {
+            SetupMocks();
+
+            var result = await _classUnderTest.ExecuteAsync(_useCaseRequest).ConfigureAwait(true);
+
+            result[0].Id.Should().Be(_documentSubmission1.Id);
```

Comment:
To help with readability you could refactor `result[0]` and assign it to `DocumentSubmissionResponse documentSubmissionResponse1`, same for `result[1]`